roles/tarzan/tasks/main.yml

```diff
@@ -16,9 +16,20 @@
               backup=no
   tags: tarzan
 
+- name: add the lua_path to funk_004's environment via the sourceme_upps script
+  lineinfile: dest={{ ngi_pipeline_conf }}/{{ bash_env_upps_script }}
+              line='export LUA_PATH={{ lua_path }}'
+              backup=no
```

Comment:
In what way do you mean? We do add env variables for ngi_pipeline and charon "In the same way" as in adding common variables to the common sourceme script, and site specific variables to the site specific sourcme script.